watertap/unit_models/zero_order/tests/test_magprex_zo.py

```diff
@@ -0,0 +1,370 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Tests for zero-order Magprex reactor model for struvite precipitation
+"""
+import pytest
+import os
+
+from pyomo.environ import (
+    Block,
+    ConcreteModel,
+    Constraint,
+    value,
+    Var,
+    assert_optimal_termination,
+)
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.solvers import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+from idaes.core import UnitModelCostingBlock
+
+from watertap.unit_models.zero_order import MagprexZO
+from watertap.core.wt_database import Database
+from watertap.core.zero_order_properties import WaterParameterBlock
+from watertap.core.zero_order_costing import ZeroOrderCosting
+
+solver = get_solver()
+
+
+class TestMagprexZO:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.db = Database()
+
+        m.fs = FlowsheetBlock(dynamic=False)
+        m.fs.params = WaterParameterBlock(
+            default={"solute_list": ["phosphates", "struvite"]}
```

Comment:
```suggestion
            solute_list=["phosphates", "struvite"]
```
I'm not sure how many other places you have used `default` in this PR, but you might consider removing them in order to prevent deprecation warnings from IDAES.

Reply:
Good catch, this one must've slipped through the cracks, but I'll double check to see if I got all of them